pkgs/os-specific/linux/syslinux/default.nix

```diff
@@ -66,6 +66,7 @@ stdenv.mkDerivation {
     "DATADIR=$(out)/share"
     "MANDIR=$(out)/share/man"
     "PERL=perl"
+    "HEXDATE=0x5e71e89e" # hardcode HEXDATE to improve reproducibility
```

Comment:
What's the meaning behind 0x5e71e89e?

Reply:
It's was current time as of making this pull request, I could put something else, although worth noting the value is used for disk verification, so 0x00000000 is non-ideal, but maybe I should use such a value.